dist/Lob-API-bundled.yml

```diff
@@ -1062,6 +1062,10 @@ components:
       oneOf:
         - $ref: '#/components/schemas/inline_address_us'
         - $ref: '#/components/schemas/inline_address_intl'
+    local_file_path:
+      type: string
+      description: A link to a remote file to be used for the postcard template.
+      pattern: '[a-zA-Z0-9@:%._+~#=/]{1,256}.(html|pdf)+$'
```

Comment:
is this really the constraint? like no `.htm` files?

Reply:
Hmm, the docs say HTML files. It didn't strike me to include `htm` but I don't see why it's unacceptable... I can try and upload an HTM file in staging to check.

Reply:
Just checked and the docs do word it as "a local upload of an HTML or PDF file" with no mention of HTM files. What's even weirder is that when I download an HTML template STRAIGHT from [the gallery linked in the docs](https://www.lob.com/template-gallery) and then navigate to the Postcards dashboard to create a test postcard, I'm unable to upload the template, regardless of whether its suffix is `html` or `htm`. Is this a bug of some kind? @hilary

Reply:
Does `htm` work via `curl`?

Reply:
Filed a bug ticket with Print & Mail API.

Reply:
Addendum: HTML & HTM both work with `curl`, so I'll add HTM to the list of extensions.